e2e/tests/nightly/admin/user-admin-ui.spec.ts

```diff
@@ -151,8 +69,7 @@ describe('User Admin', () => {
     const userNameAuditPage = await userAuditPage.getUsernameValue();
     expect(userNameAuditPage).toEqual(userNameValue);
 
-    await userAuditPage.clickUserAdminLink();
-    const userProfileInfo = new UserProfileInfo(page);
```

Comment:
this was incorrect because there are 2 open pages at this point: the all-users admin page opens a new tab for User Audit.  This `page` was referring to the (all-users) admin page but it needed to refer to the audit page instead.  Now the audit page creates the new profile page, and can supply `page`.

Did all that make sense?